Source/WebCore/platform/graphics/SourceBufferPrivate.cpp

```diff
@@ -54,6 +54,8 @@ namespace WebCore {
 // new current time without triggering this early return.
 // FIXME(135867): Make this gap detection logic less arbitrary.
 static const MediaTime discontinuityTolerance = MediaTime(1, 1);
+static const unsigned evictionAlgorithmInitialTimeChunk = 30000;
+static const unsigned evictionAlgorithmTimeChunkLowThreshold = 3000;
```

Comment:
Why milliseconds? Why not MediaTime? It's going to get converted to MediaTime anyway.

Reply:
I guess at the moment of coding it, it was easier to divide by two. I could try to get it back to MediaTime.

Reply:
Now that I checked, MediaTimes does not have support for any `operator/` so I could not use it unless I extracted the value from the MediaTime, divide it and recreate it. I find current solution better.

Reply:
That is baffling to me as dividing MediaTimes should be trivial (just multiply the denominator). I'll have a look.

Reply:
It is for fractions, maybe not so easy for doubles.

Reply:
What would be the problem for doubles? Division in IEEE 754 handles division by zero even, so I can imagine it being a matter of dividing doubles and updating MediaTime flags.

Yesterday I started writing a patch for adding division to MediaTime (didn't get very far because it was late and soon got busy with other stuff), so I'm particularly interested in potential problems.

Reply:
Sure, but I didn't think necessary to write it, just avoided the fuzz.

Reply:
Oh yeah. You don't need to go into that rabbit hole for this patch. We have someone in-team for that. :rabbit2::raising_hand_woman: